src/Tizen.NUI/src/internal/Interop/Interop.Accessibility.cs

```diff
@@ -32,6 +32,9 @@ internal static partial class Accessibility
 
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "csharp_dali_accessibility_stop_reading")]
             public static extern void StopReading(global::System.Runtime.InteropServices.HandleRef jarg1, bool jarg2);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "csharp_dali_accessibility_supress_screen_reader")]
```

Comment:
This looks related to csharp-binder patch. could you share the dali-csharp-binder's patch link?

Reply:
dali-csharp-binder patch:
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/257089/